### DISCUSSION TITLE
Is there a way to move point constraint points?

### DISCUSSION THREAD

**Discussion by tksuoran** [jrouwe/JoltPhysics#255] [Q&A]
I could not find a way to move point constraint points after constraint has been created. I hacked together #253 and it seems to work for me.

**Comment by jrouwe** [comment]
There is indeed no way except for destroying and recreating the constraint at the moment.

**Comment by tksuoran** [comment]
The point constraint in action: https://twitter.com/tksuoran/status/1579392952862113792?s=20&t=XuRRDTF5Qvra5naw9Gt9MA
